utils/makefile.sphinx

```diff
@@ -1,20 +1,16 @@
 # -*- coding: utf-8; mode: makefile-gmake -*-
 
-export DOCS_FOLDER DOCS_BUILD DOCS_DIST BOOKS_FOLDER BOOKS_DIST
+export DOCS_FOLDER DOCS_BUILD DOCS_DIST
```

Comment:
We don't publish the .tex  / .pdf versions.
Tex requires a lot of dependencies (not remove from utils/searx.sh )

Reply:
Tex is a intermediate product of PDF generation / see https://github.com/searx/searx/pull/2530 and https://searx.github.io/searx/admin/buildhosts.html

    $ sudo -H ./utils/searx.sh install buildhost  # install Tex and more to build PDF
    $ make books/user.pdf